Source/WebCore/html/canvas/CanvasRenderingContext2DBase.cpp

```diff
@@ -1155,6 +1155,9 @@ bool CanvasRenderingContext2DBase::isPointInStroke(Path2D& path, double x, doubl
 
 bool CanvasRenderingContext2DBase::isPointInPathInternal(const Path& path, double x, double y, CanvasFillRule windingRule)
 {
+    if (!std::isfinite(x) || !std::isfinite(y))
+        return false;
```

Comment:
Please replace the check for the `transformedPoint` below:
```
  if (!std::isfinite(transformedPoint.x()) || !std::isfinite(transformedPoint.y()))
        return false;
```
with the assertion:
```
ASSERT(std::isfinite(transformedPoint.x()) && std::isfinite(transformedPoint.y());
```
If `x` and `y` are finite numbers, `transformedPoint.x()` and `transformedPoint.y()` have to be finite as well.

Reply:
Done.